coredns/README.md

```diff
@@ -6,38 +6,152 @@ Get metrics from CoreDNS in real time to visualize and monitor DNS failures and
 
 ## Setup
 
-Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][7] for guidance on applying these instructions.
-
 ### Installation
 
 The CoreDNS check is included in the [Datadog Agent][1] package, so you don't need to install anything else on your servers.
 
 ### Configuration
+<!-- xxx tabs xxx -->
+<!-- xxx tab "Docker" xxx -->
+#### Docker
 
-#### Containerized
+To configure this check for an Agent running on a container:
 
-For containerized environments, see the [Autodiscovery Integration Templates][7] for guidance on applying the parameters below.
+##### Metric Collection
 
-#### Metric Collection
+Set [Autodiscovery Integration Templates][9] as Docker labels on your application container:
```

Comment:
All the links added to this page are broken. See: https://docs-staging.datadoghq.com/cswatt/okr-previews/integrations/coredns/?tab=docker#configuration